libcontainer/nsenter/nsexec.c

```diff
@@ -713,10 +713,12 @@ void send_idmapsources(int sockfd, pid_t pid, char *idmap_src, int idmap_src_len
 		ret = sys_mount_setattr(fd_tree, "", AT_EMPTY_PATH, &attr, sizeof(attr));
 		if (ret < 0) {
 			sane_kill(pid, SIGKILL);
-			if (errno == EINVAL)
-				bail("failed to change mount attributes, maybe the filesystem doesn't supports ID-mapped mounts");
+			if (errno == ENOSYS)
+				bail("failed to use mount_setattr(2), the kernel doesn't supports ID-mapped mounts");
+			else if (errno == EINVAL)
+				bail("failed to use mount_setattr(2), maybe the filesystem doesn't supports ID-mapped mounts: %s", idmap_src);
```

Comment:
At least we should change `supports` to `support`. ^_^

Reply:
There are three `doesn't supports` need to change.

Reply:
Fixed all of them, thanks!